packages/xo-web/src/xo-app/vm-import/ova/index.js

```diff
@@ -98,9 +98,24 @@ function parseTarHeader(header) {
   if (fileName.length === 0) {
     return null
   }
-  const fileSize = parseInt(
-    Buffer.from(header.slice(124, 124 + 11)).toString('ascii'),
-    8
+  // https://stackoverflow.com/a/2511526/72637
+  const sizeBuffer = Buffer.from(header.slice(124, 124 + 12))
+  let fileSize = 0
+  if (sizeBuffer[0] === 128) {
+    // https://github.com/chrisdickinson/tar-parse/blob/master/header.js#L271
+    // remove the head byte and go from low to high power.
+    const array = [...sizeBuffer.slice(1)].reverse()
+    array.forEach((value, index) => {
+      fileSize += value * Math.pow(256, index)
+    })
+  } else fileSize = parseInt(sizeBuffer.slice(0, -1).toString('ascii'), 8)
+  console.log(
```

Comment:
Is this log necessary/wanted in production?

Reply:
I liked it, I was wondering what was the corporate policy around console.log()

Reply:
I try to avoid it unless it's necessary to debug in production.

Reply:
we've had issue where we had an error message reported and absolutely no clue what happened.

Reply:
Because this is a specific process, we'll keep it.